front/src/reducers/osrdconf/infra_schema.json

```diff
@@ -1593,7 +1593,7 @@
             "type": "object"
         },
         "TrackSection": {
-            "description": "A track section is a piece of track and is the tracking system used in OSRD to locate a point.\nA track section is identified by his unique id and its coordinates (geographic or schematic).",
```

Comment:
Can you also update the french translation ? You can find it in public/locales/fr/infraEditor.json ? You can search for "schématiques"